### PR TITLE
add __str__ method

### PR DESCRIPTION
**Pull request by YasiucheniaAndrey**

### CODE REVIEW

File: product/views.py [hunk 1]
```diff
@@ -1,6 +1,22 @@
 from django.shortcuts import render
 from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+from .models import Product
 
 
 def index(request):
-    return HttpResponse("Product page")
+    product_list = Product.objects.order_by('-id')
```
Comment (revimi) [inline product/views.py]:
```suggestion
    product_list = Product.objects.all()
```

File: product/views.py [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+    context = {
+        'product_list':product_list,
+    }
+    return render(request, 'product/index.html', context)
+
+def detail(request, product_id):
+    product = get_object_or_404(Product,pk=product_id)
+    context = {'pd': product.description,
+               'psd': product.short_description,
+               'pp': product.price}
```
Comment (revimi) [inline product/views.py]: Please use readable names for variables and dict keys

Comment (revimi) [reply product/views.py]:
```suggestion
    product = get_object_or_404(Product,pk=product_id)
    context = {'product_description': product.description,
               'product_short_description': product.short_description,
               'product_price': product.price}
```

Comment (YasiucheniaAndrey) [reply product/views.py]: done

File: product/views.py (hunk 1 above)
Comment (YasiucheniaAndrey) [reply product/views.py]: done

**Review by revimi — APPROVED** [review]

**Pull request merged**